packages/simcore-sdk/tests/integration/test_node_ports_v2_r_clone.py

```diff
@@ -0,0 +1,180 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import shutil
+from contextlib import asynccontextmanager
+from pathlib import Path
+from typing import AsyncGenerator, AsyncIterable, Iterator
+
+import aioboto3
+import pytest
+import sqlalchemy as sa
+from faker import Faker
+from pytest_mock.plugin import MockerFixture
+from settings_library.r_clone import RCloneSettings
+from simcore_postgres_database.models.file_meta_data import file_meta_data
+from simcore_sdk.node_ports_v2 import r_clone
+
+pytest_simcore_core_services_selection = [
+    "migration",
+    "postgres",
+    "storage",
+]
+
+pytest_simcore_ops_services_selection = [
+    "minio",
+    "adminer",
+]
+
+
+class _TestException(Exception):
+    pass
+
+
+# FIXTURES
+
+
+@pytest.fixture
+def file_name(faker: Faker) -> str:
+    return f"file_{faker.uuid4()}.txt"
+
+
+@pytest.fixture
+def upload_file_dir(tmpdir: Path) -> Iterator[Path]:
```

Comment:
Please check the doc of the ``tmp_path`` pytest fixture

Q: What is the different between ``tmpdir`` and ``tmp_path`` fixtures?